sdk/Sdk/FunctionMetadataGenerator.cs

```diff
@@ -804,8 +806,78 @@ private static bool SupportsDeferredBinding(CustomAttribute attribute)
                 return false;
             }
 
-            return typeDefinition.CustomAttributes
-                                 .Any(a => string.Equals(a.AttributeType.FullName, Constants.SupportsDeferredBindingAttributeType, StringComparison.Ordinal));
+            // checking attributes advertised by the binding attribute
+            foreach (CustomAttribute bindingAttribute in typeDefinition.CustomAttributes)
+            {
+                if (string.Equals(bindingAttribute.AttributeType.FullName, Constants.InputConverterAttributeType, StringComparison.Ordinal))
+                {
+                    // InputConverterAttribute will have supported converter type
+                    foreach (var customAttribute in bindingAttribute.ConstructorArguments)
+                    {
+                        if (DoesConverterSupportDeferredBinding(customAttribute, bindingType))
+                        {
+                            return true;
```

Comment:
nit: `return DoesConverterSupportDeferredBinding(customAttribute, bindingType)`

I don't think we need an if-statement here if we are returning true if DoesConverterSupportDeferredBinding is true

Reply:
That suggested change would have behavioral differences then what is here when there is multiple `InputConverterAttribute`'s set, right?

Reply:
Exactly.